.gitignore

```diff
@@ -15,6 +15,7 @@ __pycache__
 */cython_version.py
 htmlcov
 .coverage
+coverage.xml
```

Comment:
What's this?

Reply:
This file is generated in these files, using the `--cov` parameter or argument.

`tox.ini`

```
pytest --pyargs tardis {toxinidir}/docs --cov tardis --cov-config={toxinidir}/setup.cfg {posargs}
```

`.github/workflows/tests.yml`

```
PYTEST_FLAGS: --tardis-refdata=${{ github.workspace }}/tardis-refdata --tardis-regression-data=${{ github.workspace }}/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html
```

---

I added it because, out of curiosity, I ran my local PyTest to verify the coverage, and I noticed that it added this file to the untracked files in Git. If I hadn't noticed, I could have added it with the command `git add .`. It is something that someone else could do.